modules/consensus/accept_test.go

```diff
@@ -321,13 +321,13 @@ func TestInconsistentCheck(t *testing.T) {
 	// Corrupt the consensus set.
 	var sfod types.SiafundOutputID
 	var sfo types.SiafundOutput
-	for id, output := range cst.cs.siafundOutputs {
+	cst.cs.db.forEachSiafundOutputs(func(id types.SiafundOutputID, output types.SiafundOutput) {
 		sfod = id
 		sfo = output
-		break
-	}
+	})
```

Comment:
It does behave differently. Basically instead of setting sfod and sfo to the first element, it sets them to the last after iterating over the entire set. As this construction only occurs in tests and my current setdb implementation has no mechanism for escaping the loop part way through it seemed better not to dwell on it too much.